spec/features/hub/authenticate_spec.rb

```diff
@@ -81,4 +81,22 @@
     # Show our specific custom error message
     expect(page).to have_text("Oops, we're sorry, but something went wrong")
   end
+
+  scenario "non-admin user is forced to reset password" do
+    user.assign_attributes(forced_password_reset_at: nil)
```

Comment:
If you're updating one attribute on `user` it's tidier to do `user.update()`. It's also good to put that in a `before`.

I notice the next line calls `validate: false`. Is that needed? I didn't see why.

Reply:
This prevents the password strength validation from occurring so we can use a weaker password when signing in (to confirm it'll check it and encourage a redirect)